test/test_archive.py

```diff
@@ -1,6 +1,8 @@
 from lyncs_io.archive import Data, Archive, split_filename
+import pytest
 
 
+@pytest.mark.mpi_skip()
```

Comment:
I will undo this changes for two reasons:
1) I don't like the idea of adding this decorator to all serial tests
2) it is good to check if also serial tests work in a parallel environment.

I would suggest the use of this flag ONLY if the test is supposed to not work in a parallel environment.

Reply:
I thought that might be a stretch. The only reason I did that was because like that you could do the test directory in parallel and the serial tests would be skipped. In other words when you would run the parallel tests you wouldn't have to be explicit about the directory they reside in.